conf/config-default.yaml

```diff
@@ -381,6 +381,7 @@ plugins:                          # plugin list (sorted by priority)
   - rocketmq-logger                # priority: 402
   - syslog                         # priority: 401
   - udp-logger                     # priority: 400
+  #- clickhouse-logger             # priority: 399
```

Comment:
ditto

Reply:
Priority 410 - 400 all for push log via network, clickhouse-logger is  via http request to push log too. It is fewer people using than http-logger，so the priority is low than http-logger.

Reply:
sorry, I found file-logger priority is 399.